src/components/common/ForeignJurisdiction.vue

```diff
@@ -145,6 +178,7 @@ export default class ForeignJurisdiction extends Mixins(CountriesProvincesMixin)
   }
 
   /** Emits an event indicating whether or not the form is valid. */
+  @Watch('valid')
```

Comment:
Why do you need this?

Reply:
No do not need this. It was me experimenting. Removing it